prac_05/state_names.py

```diff
@@ -10,17 +10,17 @@
 print(CODE_TO_NAME)
 
 state_code = input("Enter short state: ").upper()
+# print(state_code)
 while state_code != "":
-    #     if state_code in CODE_TO_NAME:
-    #         print(state_code, "is", CODE_TO_NAME[state_code])
-    #     else:
-    #         print("Invalid short state")
-    #     state_code = input("Enter short state: ").upper()
+    # if state_code in CODE_TO_NAME:
+    #     print(state_code, "is", CODE_TO_NAME[state_code])
+    # else:
+    #     print("Invalid short state")
+    # state_code = input("Enter short state: ").upper()
     try:
-        state_code = state_code in CODE_TO_NAME
```

Comment:
Should delete this code, I think it is a bug

Reply:
okie